databuilder/extractor/neo4j_search_data_extractor.py

```diff
@@ -72,3 +77,18 @@ def extract(self):
     def get_scope(self):
         # type: () -> str
         return 'extractor.search_data'
+
+    def _add_publish_tag_filter(self, publish_tag, cypher_query):
+        """
+        Adds publish tag filter into Cypher query
+        :param publish_tag: value of publish tag.
+        :param cypher_query:
+        :return:
+        """
+        # type: (str, str) -> str
+        if not publish_tag:
+            publish_tag_filter = ''
+        else:
+            publish_tag_filter = """WHERE table.published_tag = '{}'""".format(publish_tag)
```

Comment:
do we need space in the filter?

Reply:
Good question. The filter is already placed in new line so no white space is needed